ci/appveyor/before-build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+export PATH="/mingw64/bin:/usr/bin:/usr/local/bin:$PATH"
+export PATH="/mingw64/lib:/usr/lib:/usr/local/lib:$PATH"
+export PATH="/c/Program\ Files\ (x86)/qhttpengine/lib:$PATH"
+
+cd /c/projects/keepassxc
+mkdir build && cd build
+
+if [ "${configuration}" = "Debug" ]; then
+  cmake -G "MSYS Makefiles" \
+    -DWITH_XC_AUTOTYPE=ON \
```

Comment:
You're missing a few build options here. This should be

```
cmake \
-DCMAKE_BUILD_TYPE=${configuration} \
-DWITH_XC_AUTOTYPE=ON \
-DWITH_XC_HTTP=ON \
-DWITH_XC_BROWSER=ON \
-DWITH_XC_YUBIKEY=ON \
-DWITH_XC_SSHAGENT=ON \
-DWITH_TESTS=ON \
-DWITH_GUI_TESTS=ON \
-DWITH_ASAN=ON ..
```

In TC we also have another entry in the build matrix for the compiler used:

```

-DCMAKE_C_COMPILER=%CMAKE_C_COMPILER% \
-DCMAKE_CXX_COMPILER=%CMAKE_CXX_COMPILER% \
```

Reply:
Thanks for the info.; I made some changes and now got some more useful information for debugging: https://ci.appveyor.com/project/adolfogc/keepassxc/build/job/44jbyxqkds3j6lhi

Reply:
It may be related; the 29th test fails with a segfault on macOS:
```
29/30 Test #29: testgui ..........................***Exception: SegFault 19.68 sec
```